.agignore

```diff
@@ -0,0 +1 @@
+custom
```

Comment:
It doesn't belong. I took measures to make sure this doesn't happen in the future.